docs/hub/model-card-landscape-analysis.md

```diff
@@ -24,12 +24,12 @@ These groupings are not mutually exclusive; they do include overlapping aspects
 | DATA                             	| ***Data Cards*** [(Pushkarna et al., 2021)](https://arxiv.org/pdf/2204.01075.pdf)                                                                                  	| “Data Cards are structured summaries of essential facts about various aspects of ML datasets needed by stakeholders across a dataset’s lifecycle for responsible AI development. These summaries provide explanations of processes and rationales that shape the data and consequently the models.”                                                                                                  	| See the [Data Cards Playbook github](https://github.com/PAIR-code/datacardsplaybook/)                                                                                            	|
-| DATA                             	| ***CrowdWorkSheets***  [(Díaz et al., 2022)](https://facctconference.org/static/pdfs_2022/facct22-181.pdf)                                                         	| “We introduce a novel framework, CrowdWorkSheets, for dataset developers to facilitate transparent documentation of key decisions points at various stages of the data annotation pipeline: task formulation, selection of annotators, plat- form and infrastructure choices, dataset analysis and evaluation, and dataset release and maintenance.”                                                 	| See [(Díaz et al., 2022)](https://facctconference.org/static/pdfs_2022/facct22-181.pdf)                                                                                          	|
+| DATA                             	| ***CrowdWorkSheets***  [(Díaz et al., 2022)](https://arxiv.org/pdf/2206.08931.pdf)                                                         	| “We introduce a novel framework, CrowdWorkSheets, for dataset developers to facilitate transparent documentation of key decisions points at various stages of the data annotation pipeline: task formulation, selection of annotators, plat- form and infrastructure choices, dataset analysis and evaluation, and dataset release and maintenance.”                                                 	| See [(Díaz et al., 2022)](https://arxiv.org/pdf/2206.08931.pdf)                                                                                          	|
```

Comment:
The arxiv links could also be hf.co/papers links

Reply:
I literally thought that was I pushing 
I'll update them now